tests/integration/test_render.py

```diff
@@ -194,3 +202,21 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_inactive(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index, zone in enumerate(zones):
+            elt = 'zone-{}'.format(index + 1)
```

Comment:
@agaylard To make this work as an input for `_get_style`, do:

```python
selector = `#zone-{}'.format(index+1)
```

This should enable `$("{selector}")` in `_get_style` to pick up the correct element.